### PR TITLE
LISAv2 : added default tags to Azure resource group.

### PR DESCRIPTION
**Pull request by iamshital**
Following tags are added to the every new resource group.

1. BuildUser
2. BuildURL
3. TestName
4. BuildMachine
5. CreationTime

### CODE REVIEW

**Comment by iamshital** [comment]
**Execution verified on local.**
```
10/19/2018 11:31:20 : [INFO ] ICA-RG-SingleVM-PERSISTANT-FMOX-636755202796 does not exists.
10/19/2018 11:31:20 : [INFO ] Using location : southcentralus
VERBOSE: Performing the operation "Replacing resource group ..." on target "ICA-RG-SingleVM-PERSISTANT-FMOX-636755202796".
VERBOSE: 4:31:22 AM - Created resource group 'ICA-RG-SingleVM-PERSISTANT-FMOX-636755202796' in location 'southcentralus'
10/19/2018 11:31:23 : [INFO ] Resource Group ICA-RG-SingleVM-PERSISTANT-FMOX-636755202796 Created.
10/19/2018 11:31:23 : [INFO ] Setting ICA-RG-SingleVM-PERSISTANT-FMOX-636755202796 tag : BuildUser = Shital
10/19/2018 11:31:26 : [INFO ] Setting ICA-RG-SingleVM-PERSISTANT-FMOX-636755202796 tag : BuildURL = NA
10/19/2018 11:31:29 : [INFO ] Setting ICA-RG-SingleVM-PERSISTANT-FMOX-636755202796 tag : TestName = VERIFY-DEPLOYMENT-PROVISION
10/19/2018 11:31:32 : [INFO ] Setting ICA-RG-SingleVM-PERSISTANT-FMOX-636755202796 tag : BuildMachine = V-SHISAV-PC\V-SHISAV-PC
10/19/2018 11:31:35 : [INFO ] Setting ICA-RG-SingleVM-PERSISTANT-FMOX-636755202796 tag : CreationTime = 2018/10/19 11:31:35
10/19/2018 11:31:39 : [INFO ] [Attempt 1/999] : Getting Existing Storage account information...
```

**Jenkins is pending.**

File: Libraries/Azure.psm1 [hunk 1]
```diff
@@ -2770,4 +2771,87 @@ Function Set-SRIOVinAzureVMs {
...
+            $hash = $ExistingTags
+            if ( -not $ExistingKeyUpdated) {
+                $hash.Add($TagName,$TagValue)
+            }
+        }
+        else {
+            $hash = @{}
+            $hash.Add($TagName,$TagValue)
+        }  
+        $Null = Set-AzureRmResourceGroup -Name $ResourceGroup -Tag $hash
```
Comment (ader1990) [inline Libraries/Azure.psm1]: you can use cmd | Out-Null, otherwise psscryptanalyzer will complain

**Comment by juhlee-microsoft** [comment]
Will work on the same change in another PR soon. But do not close this PR

**Comment by juhlee-microsoft** [comment]
#311 passed the unit test and merged to master. Close this PR.

**Pull request closed without merging**